src/site/content/en/blog/aspect-ratio/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: 'Maintaining Aspect Ratio in CSS'
+subhead: The new aspect-ratio property can help with responsive dimension-based imaged.
+authors:
+  - una
+date: 2021-01-14
+hero: hero.jpg
+alt: Image of colorful telephones resized with the same aspect ratio.
+description: There are several ways to maintain aspect ratio and image dimensions on the web. This post goes over two ways—one popular hack, and one new CSS property.
+tags:
+  - blog
+  - css
+---
```

Comment:
The above-the-fold experience currently doesn't provide a very good summary of the actual contents of this page. The `description` that you provided above is the most accurate IMO. The post covers the old hack, and the new property. Notice that this above-the-fold doesn't mention anywhere that you cover the old hack.

![image](https://user-images.githubusercontent.com/4713486/104794638-bb3f6080-575d-11eb-8cf1-f7f5355d6c44.png)

My request here is for you to somehow update this above-the-fold experience to actually tell the reader what they will get out of this post (hint: the `subhead` and `description` can be duplicate)

Reply:
See also my recent email to the team about not burying the lede: https://groups.google.com/a/google.com/g/web-devrel-team/c/O5xnGH6JeMQ

Reply:
Yes these are good changes, definitely changing up a few things.